packages/core/ts/MobileCore.ts

```diff
@@ -43,7 +43,7 @@ const RCTAEPCore: IMobileCore = NativeModules.AEPCore;
 const MobileCore: IMobileCore = {
   /**
    * Returns the version of the AEPCore extension
-   * @param  {string} Promise a promise that resolves with the extension version
+   * @return  {string} Promise a promise that resolves with the extension version
```

Comment:
* @return {string} Promise that resolves with the extension version

Reply:
`{string} Promise`  is the return type, and the following `a promise ...` is the description. I think we'd better keep the current description.